R/get_watershed_WFS.R

```diff
@@ -17,21 +17,24 @@
 #'
 #'@import sbtools 
 #'@export
-get_watershed_WFS = function(site, session = NULL){
+get_watershed_WFS = function(site, ...){
   
-  watershed_item <- get_watershed_item(site, session)
+  watershed_item <- get_watershed_item(site, ...)
  
   WFS_url <- match_url_distro(watershed_item, "ScienceBase WFS Service")
   return(WFS_url)
 }
 
-get_watershed_item = function(site, session){
+get_watershed_item = function(site, ...){
+	
+	session_check_reauth(...)
+	
   # session not needed since items are public
-  identifier <- query_item_identifier(scheme = get_scheme(), type = 'watershed', key = site,  session = session)
+  identifier <- query_item_identifier(scheme = get_scheme(), type = 'watershed', key = site)
```

Comment:
Talking with Jordan. We should assume auth is needed. That comment should be changed (and was contradictory before as we were passing in a session).